geode-core/src/distributedTest/java/org/apache/geode/management/JMXMBeanReconnectDUnitTest.java

```diff
@@ -135,10 +139,12 @@ public void setUp() throws Exception {
     locator2JmxPort = port[3];
     locators = "localhost[" + locator1Port + "];localhost[" + locator2Port;
 
-    locator1VM.invoke(
-        () -> startLocator(locator1Name, locator1Dir, locator1Port, locator1JmxPort, locators));
-    locator2VM.invoke(
-        () -> startLocator(locator2Name, locator2Dir, locator2Port, locator2JmxPort, locators));
+    locator1VM.invoke(() -> {
+      startLocator(locator1Name, locator1Dir, locator1Port, locator1JmxPort, locators);
+    });
+    locator2VM.invoke(() -> {
+      startLocator(locator2Name, locator2Dir, locator2Port, locator2JmxPort, locators);
```

Comment:
Just curious, why did switch from the short form to the expanded form for the lambda

Reply:
Just to avoid line-break in the `startLocator` line. It was an impulse and I don't feel strongly about it.

Reply:
Cool, just trying to figure out if I could do something better.